src/Tabs/RoomTab.as

```diff
@@ -15,11 +15,15 @@ class RoomTab : Tab {
     bool isEditing = true;
     bool isActive = true;
 
+    int get_WindowFlags() override {
+        return UI::WindowFlags::NoCollapse;
+    }
```

Comment:
didnt want autoresize since I wanted to be able to move and see multiple windows on one screen